systemtest/src/test/java/io/strimzi/systemtest/mirrormaker/MirrorMaker2ST.java

```diff
@@ -1069,6 +1075,499 @@ void testRestoreOffsetsInConsumerGroup(ExtensionContext extensionContext) {
+        kafkaClientsPodName = kubeClient().listPodsByPrefixInName(testStorage.getKafkaClientsName()).get(0).getMetadata().getName();
+
+        internalKafkaClient = internalKafkaClient.toBuilder()
+            .withUsingPodName(kafkaClientsPodName)
+            .withTopicName(topicSourceNameB)
+            .withClusterName(kafkaClusterSourceName)
+            .withKafkaUsername(userSource.getMetadata().getName())
+            .withListenerName(kafkaTlsScramListenerName)
+            .build();
+
+        LOGGER.info("[SCRAM-SHA] Sending messages to - topic {}, cluster {} and message count of {}.",
+            topicSourceNameB, kafkaClusterSourceName, messagesCount);
+        sent = internalKafkaClient.sendMessagesTls();
+
+        LOGGER.info("[SCRAM-SHA] Changing to target - topic {}, cluster {}, user {} for next internalKafkaClient.",
+                topicTargetNameB, kafkaClusterTargetName, userTarget.getMetadata().getName());
+        internalKafkaClient = internalKafkaClient.toBuilder()
+            .withTopicName(topicTargetNameB)
+            .withClusterName(kafkaClusterTargetName)
+            .withKafkaUsername(userTarget.getMetadata().getName())
+            .withConsumerGroupName(ClientUtils.generateRandomConsumerGroup())
+            .build();
+
+        LOGGER.info("[SCRAM-SHA] Now messages should be mirrored to target topic and cluster");
+        internalKafkaClient.consumesTlsMessagesUntilOperationIsSuccessful(sent, Duration.ofMinutes(10).toMillis());
+        LOGGER.info("[SCRAM-SHA] Messages successfully mirrored");
+    }
+
+    @ParallelNamespaceTest
+    @Tag(ACCEPTANCE)
+    @SuppressWarnings({"checkstyle:MethodLength"})
+    void testKMM2RollAfterSecretsCertsUpdateTLS(ExtensionContext extensionContext) throws Exception {
+        final String namespaceName = StUtils.getNamespaceBasedOnRbac(INFRA_NAMESPACE, extensionContext);
+        final String kafkaClientsName = mapWithKafkaClientNames.get(extensionContext.getDisplayName());
+        String clusterName = mapWithClusterNames.get(extensionContext.getDisplayName());
+        String kafkaClusterSourceName = clusterName + "-source";
+        String kafkaClusterTargetName = clusterName + "-target";
+        String topicName = "availability-topic-source-" + mapWithTestTopics.get(extensionContext.getDisplayName());
+        String topicSourceNameMirrored = kafkaClusterSourceName + "." + topicName;
+        String topicSourceNameA = MIRRORMAKER2_TOPIC_NAME + "-" + rng.nextInt(Integer.MAX_VALUE);
+        String topicTargetNameA = kafkaClusterSourceName + "." + topicSourceNameA;
+        String topicSourceNameB = MIRRORMAKER2_TOPIC_NAME + "-" + rng.nextInt(Integer.MAX_VALUE);
+        String topicTargetNameB = kafkaClusterSourceName + "." + topicSourceNameB;
+        String kafkaUserSourceName = clusterName + "-my-user-source";
+        String kafkaUserTargetName = clusterName + "-my-user-target";
```

Comment:
In previous testcase you are using `TestStorage` and here not, does it make sense to use it here as well?